src/olympia/reviewers/templates/reviewers/base.html

```diff
@@ -16,13 +16,13 @@ <h1 class="site-title amo">
     {% if action_allowed(amo.permissions.ADDONS_REVIEW) and action_allowed(amo.permissions.THEMES_REVIEW) %}
       <nav class="pill-nav-amo">
         {% if action_allowed(amo.permissions.ADDONS_REVIEW) %}
-          <a{% if '/reviewers/themes' not in request.path %} class="selected"{% endif %}
+          <a{% if '/editors/themes' not in request.path %} class="selected"{% endif %}
```

Comment:
haha, ugh. What a terrible `if`.  Ack'd its just a bug fix and you didn't write it... but is there no better way of doing it?  E.g. checking the view name (request has the view name, right?) seems less fragile.

Reply:
Yes, absolutely. The trick however is that it needs to work for all views that are inside that path, so you need to name the views in a coherent way and check that...

...but it's going to go away soon when I rewrite the reviewers dashboard (one of the tasks is to change the navigation completely, getting rid of that link) so I didn't want to bother with that just now.